tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -14,7 +14,7 @@ describe('Header', () => {
     auth: {
       user: {
         email: 'test@test.com',
-        roles: [{ id: 1, name: 'OFA System Admin', permissions: [] }],
+        roles: [{ id: 1, name: 'Developer', permissions: [] }],
```

Comment:
this is the only role that's suppose to have both permission to data file and permission to admin

Reply:
open to more tests, though we are still covered over 90% according to code cov